_core/smart/tutorial.md

```diff
@@ -1,211 +1,188 @@
 
-Note that we're importing modules from the `@blockstack/clarity` package:
+As you can see on the screen, every smart contract's source code is publicly verifiable through the explorer.
 
-```js
-import { Client, Provider, ProviderRegistry, Result } from "@blockstack/clarity";
-```
 
-### Initializing a client
+## Step 4: Call the public method
 
-At the test start, we are initializing a contract instance `helloWorldClient` and a provider that simulates interactions with the Stacks 2.0 blockchain. If this were in a production environment, the contract instance would be the equivalent of a contract deployed to the blockchain. The provider would be the Stacks blockchain.
+Go back to the Sandbox screen and switch to the ["Contract call"](https://testnet-explorer.blockstack.org/sandbox?tab=contract-call) tab. In the form, enter the following details:
 
-```js
-let helloWorldClient: Client;
-let provider: Provider;
+* **Contract address**: Your generated STX address. Hover over the identity component on the right side of the screen to see your full address.
+* **Contract name**: hello-world-contract
 
-(...)
+![sandbox-call](images/sandbox-call.png)
 
-provider = await ProviderRegistry.createProvider();
-helloWorldClient = new Client("SP3GWX3NE58KXHESRYE4DYQ1S31PQJTCRXB3PE9SB.hello-world", "hello-world", provider);
-```
+Verify your input and hit "Fetch" to look up available methods. You should see the 3 public methods implemented in the smart contract.
 
-Take a look at the client initialization. It requires a contract id and name in the following format: `{owner_stacks_address}.{contract_identifier}`. The second field indicates the location of the smart contract file, without the `.clar` suffix. By default, the location is assumed to be relative to the `contracts` folder.
+Locate the `(echo-number)` method, provide any integer for the `val` argument, set the post condition mode to "Allow" and hit **Submit**. A result confirmation will be displayed, indicating that a new contract call transaction was issued. Click on the transaction to review it. In the next seconds, your contract call should be completed and you will see a contract call success screen. Scroll down to the function summary section to verify your contract call:
 
-As you can see above, a sample Stacks address and contract identifier is already provided for you. You don't need to modify anything.
+![sandbox-call](images/sandbox-calltx.png)
 
-### Checking syntax
+**Congratulations! You just deployed your smart contract and called a public function on the Testnet.**
 
-Next, we check the contract for valid syntax. If the smart contract implementation has syntax error (bugs), this check would fail:
+## Step 5: Get familiar with CLI (optional)
```

Comment:
This chapter seems misplaced now. It feels like we haven't figured out a good developer journey yet. VSCode, terminal (CLI + generator), and explorer are all introduced at the same time. It feels overwhelming.

Would push the CLI intro to the counter tutorial.

Reply:
It might be even better to have a separate CLI topic, but for the moment I think covering our deployment options right here makes the most sense until we can peel that off.